migration/testdata/v1_StorageInfo.post

```diff
@@ -1,4 +1,4 @@
+/ffs/scheduler/cistore_v2/ID2/QmbtfAvVRVgEa9RH6vYpKg1HWbBQjxiZ6vNG1AAvt3vyfR,{"APIID":"ID2","JobID":"a84dedc4-b1e5-4798-b4ad-44cc803211ab","Cid":{"/":"QmbtfAvVRVgEa9RH6vYpKg1HWbBQjxiZ6vNG1AAvt3vyfR"},"Created":"2020-10-07T13:58:24.420077464Z","Hot":{"Enabled":false,"Size":0,"Ipfs":{"Created":"0001-01-01T00:00:00Z"}},"Cold":{"Enabled":true,"Filecoin":{"DataCid":{"/":"QmbtfAvVRVgEa9RH6vYpKg1HWbBQjxiZ6vNG1AAvt3vyfR"},"Size":33554432,"Proposals":[{"DealID":0,"PieceCid":{"/":"baga6ea4seaqaneh2acoyumk6krvkatezejopmpftygl6ducgrkl7qz7vnrlx2ni"},"Renewed":false,"Duration":519572,"StartEpoch":132466,"Miner":"t018780","EpochPrice":3125000},{"DealID":0,"PieceCid":{"/":"baga6ea4seaqaneh2acoyumk6krvkatezejopmpftygl6ducgrkl7qz7vnrlx2ni"},"Renewed":false,"Duration":520884,"StartEpoch":132466,"Miner":"t022352","EpochPrice":3125000000}]}}}
+/ffs/scheduler/cistore_v2/ID2/QmZTMaDfCMWqhUXYDnKup8ctCTyPxnriYW7G4JR8KXoX5M,{"APIID":"ID2","JobID":"35a6f8fc-cad5-41f4-a410-e5c3d882f0e0","Cid":{"/":"QmZTMaDfCMWqhUXYDnKup8ctCTyPxnriYW7G4JR8KXoX5M"},"Created":"2020-10-20T12:29:12.996702455Z","Hot":{"Enabled":false,"Size":0,"Ipfs":{"Created":"0001-01-01T00:00:00Z"}},"Cold":{"Enabled":true,"Filecoin":{"DataCid":{"/":"QmZTMaDfCMWqhUXYDnKup8ctCTyPxnriYW7G4JR8KXoX5M"},"Size":67108864,"Proposals":[{"DealID":0,"PieceCid":{"/":"baga6ea4seaqhvlh3ktd4rxnbozupun776lu4wh6bprrcr6t63ec2fvkkwatn4oi"},"Renewed":false,"Duration":521469,"StartEpoch":168927,"Miner":"f010507","EpochPrice":6250000000},{"DealID":0,"PieceCid":{"/":"baga6ea4seaqhvlh3ktd4rxnbozupun776lu4wh6bprrcr6t63ec2fvkkwatn4oi"},"Renewed":false,"Duration":519893,"StartEpoch":168927,"Miner":"f022142","EpochPrice":12500000},{"DealID":0,"PieceCid":{"/":"baga6ea4seaqhvlh3ktd4rxnbozupun776lu4wh6bprrcr6t63ec2fvkkwatn4oi"},"Renewed":false,"Duration":520113,"StartEpoch":149621,"Miner":"f03491","EpochPrice":6250000000}]}}}
+/ffs/scheduler/cistore_v2/ID3/QmZTMaDfCMWqhUXYDnKup8ctCTyPxnriYW7G4JR8KXoX5M,{"APIID":"ID3","JobID":"35a6f8fc-cad5-41f4-a410-e5c3d882f0e0","Cid":{"/":"QmZTMaDfCMWqhUXYDnKup8ctCTyPxnriYW7G4JR8KXoX5M"},"Created":"2020-10-20T12:29:12.996702455Z","Hot":{"Enabled":false,"Size":0,"Ipfs":{"Created":"0001-01-01T00:00:00Z"}},"Cold":{"Enabled":true,"Filecoin":{"DataCid":{"/":"QmZTMaDfCMWqhUXYDnKup8ctCTyPxnriYW7G4JR8KXoX5M"},"Size":67108864,"Proposals":[{"DealID":0,"PieceCid":{"/":"baga6ea4seaqhvlh3ktd4rxnbozupun776lu4wh6bprrcr6t63ec2fvkkwatn4oi"},"Renewed":false,"Duration":521469,"StartEpoch":168927,"Miner":"f010507","EpochPrice":6250000000},{"DealID":0,"PieceCid":{"/":"baga6ea4seaqhvlh3ktd4rxnbozupun776lu4wh6bprrcr6t63ec2fvkkwatn4oi"},"Renewed":false,"Duration":519893,"StartEpoch":168927,"Miner":"f022142","EpochPrice":12500000},{"DealID":0,"PieceCid":{"/":"baga6ea4seaqhvlh3ktd4rxnbozupun776lu4wh6bprrcr6t63ec2fvkkwatn4oi"},"Renewed":false,"Duration":520113,"StartEpoch":149621,"Miner":"f03491","EpochPrice":6250000000}]}}}
```

Comment:
Since _StorageInfo_ changed, a previous test of v1 migration had to change a bit the data.
I'll recommend that if anyone is still behind the v1 migration, jump to that one first, and then to this one.

Making things jump for v0 to v2 for the amount of things that changed would take much more work. Mostly, replication old struct versions for migrations. (and we have some long nested ones).